test/scenarios/synapse/output/coredefault/src/azure-cli/requirements.py3.Darwin.txt

```diff
@@ -127,4 +127,4 @@ wcwidth==0.1.7
 websocket-client==0.56.0
 wrapt==1.11.2
 xmltodict==0.12.0
-azure-mgmt-synapse==1.0.0
```

Comment:
Can you cherry pick this commit https://github.com/Azure/autorest.az/pull/796/commits/91a64f29095c2e073b4b33d7e1ce7dd7a1525f26 to resolve the python SDK new release version impact on scenario test ? 
as It's really not a good solution to bump python sdk version in our scenario test each time they get new releases

Reply:
Hi, will that PR be merged soon? I think this PR can wait after that was reviewed and merged.

Reply:
You need to change this change back as commit https://github.com/Azure/autorest.az/commit/91a64f29095c2e073b4b33d7e1ce7dd7a1525f26 has been merged.

Reply:
Hi @qiaozha , merged to latest master